src/main/java/eccomapp/dao/OrderDao.java

```diff
@@ -0,0 +1,90 @@
+package eccomapp.dao;
+
+import eccomapp.controller.OrderController;
+import eccomapp.entity.OrderEntity;
+import eccomapp.exception.InvalidInputException;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.UUID;
+import java.util.logging.Logger;
+
+public class OrderDao {
+    /** This method add the order to order database
+     *
+     * @param connection for connection
+     * @param name for product name
+     * @param number for mobile number
+     * @throws InvalidInputException
+     */
+    public void addOrder(Connection connection,String name,String number) throws InvalidInputException {
+        try {
+            OrderController orderController = new OrderController();
+            UserDao userDao = new UserDao();
+            UUID id = userDao.getID(connection, number);
+            ProductDao productDao = new ProductDao();
+            OrderEntity orderEntity = new OrderEntity();
+            float totalCost = productDao.getTotalCostOrder(connection, name);
+            String sql = "INSERT INTO orders VALUES(?, ?, ?,?,CURRENT_TIMESTAMP,CURRENT_TIMESTAMP)";
+            PreparedStatement statement = connection.prepareStatement(sql);
+            statement.setObject(1, orderEntity.getOrderid());
+            statement.setObject(2, id);
+            statement.setString(3, name);
+            statement.setFloat(4, totalCost);
+            statement.executeUpdate();
+            System.out.println("Total value of order is " + totalCost);
+        }
+        catch (SQLException e)
+        {
+            throw new InvalidInputException(400,"invalid input");
+        }
+    }
+
+    /** This method display the products
+     *
+     * @param connection for connection
+     * @param logger for logging
+     * @throws InvalidInputException
+     */
+    public void display(Connection connection, Logger logger) throws InvalidInputException {
+        try {
+            String sql = "SELECT * FROM products ";
```

Comment:
orderDao should have queries related to only order table. Restructure your project following this for all the dao's